src/python/tests/core/bot/tasks/corpus_pruning_task_test.py

```diff
@@ -157,11 +157,23 @@ def setUp(self):
     BaseTest.setUp(self)
     helpers.patch(self, [
         'build_management.build_manager.setup_build',
+        'base.utils.get_application_id',
+        'google_cloud_utils.big_query.get_api_client',
     ])
     self.mock.setup_build.side_effect = self._mock_setup_build
+    self.mock.get_application_id.return_value = 'project'
 
   def test_prune(self):
     """Basic pruning test."""
+    underlying = mock.MagicMock()
```

Comment:
Are these required at all? I thought return_value would default to a MagicMock, but didn't confirm.

Reply:
It turns out not all of it was necessary, but I could not get the tests to pass without the code I left.